ext/rbs_extension/extconf.rb

```diff
@@ -1,4 +1,4 @@
 require 'mkmf'
 $INCFLAGS << " -I$(top_srcdir)" if $extmk
-$CFLAGS += " -std=c99 -Wold-style-definition"
+$CFLAGS += " -std=c99 -Wold-style-definition" unless have_macro('_MSC_VER')
```

Comment:
Not all compilers other than MSVC accepts gcc options.

```suggestion
%w"-std=c99 -Wold-style-definition".each {|flag| append_cflags(flag)}
```